Cython/Compiler/Symtab.py

```diff
@@ -1964,6 +1970,11 @@ def declare_var(self, name, type, pos,
                     cname = None, visibility = 'private',
                     api = 0, in_pxd = 0, is_cdef = 0):
         if is_cdef:
+            type_type = None
+            # Slight hack: Sometimes declare_var is called before builtin_scope() is done being declared.
```

Comment:
This is probably the case while the `Builtin` module is initialising. The extension types defined in that module are all builtins and do not have metaclasses.